eng/common/scripts/Helpers/Metadata-Helpers.ps1

```diff
@@ -80,3 +80,54 @@ function GetPrimaryCodeOwner ([string]$TargetDirectory)
     Write-Warning "No code owner found in $TargetDirectory."
     return $null
 }
+
+function GetDocsMsService($clientPackageInfo, $mgmtPackageInfo, $serviceName) 
+{
+  $service = $serviceName.ToLower().Replace(' ', '').Replace('/', '-')
+  if ($clientPackageInfo -and $clientPackageInfo[0].MSDocService) {
```

Comment:
Why is clientPackageInfo an array?

Reply:
clientPackageInfo includes all packages for the service

Reply:
If it is an array why are we blindly taking the first one? We might want to pass in a single entry if we know which one is interesting. If this does remain to be an array we should change it to clientPackageInfos to make it plural.

Reply:
We can only have one ms.service per service level readme. 
`key-vault-keys, key-vault-secret, etc` share the same ms.service `key-vault`. We do not have enough information which is the best. Just take the first one. If this is not correct, we can make changes to csv. But it is supposed to be the same for packages under service.

Reply:
I got what you mean, I will pass one entry in.